equality_test.go

```diff
@@ -2,6 +2,7 @@ package assert_test
 
 import (
 	"errors"
+	m "math"
```

Comment:
I can't find any reference to `math` package in the testing code. Is this needed?

Reply:
true

Reply:
it is not used anymore, right. Can get rid of it.